sermant-plugins/sermant-router/spring-router-plugin/src/main/java/io/sermant/router/spring/utils/SpringRouterUtils.java

```diff
@@ -50,6 +51,57 @@ public class SpringRouterUtils {
     private SpringRouterUtils() {
     }
 
+    /**
+     * get http parameter
+     *
+     * @param obj HttpServletRequest
+     * @return parameter
+     */
+    public static Map<String, String[]> getParameterMap(Object obj) {
+        return (Map<String, String[]>) ReflectUtils.invokeWithNoneParameter(obj, "getParameterMap");
+    }
+
+    /**
+     * get http uri
+     *
+     * @param obj HttpServletRequest
+     * @return uri
+     */
+    public static String getRequestUri(Object obj) {
+        return ReflectUtils.invokeWithNoneParameterAndReturnString(obj, "getRequestURI");
+    }
+
+    /**
+     * get http method
+     *
+     * @param obj HttpServletRequest
+     * @return method
+     */
+    public static String getMethod(Object obj) {
+        return ReflectUtils.invokeWithNoneParameterAndReturnString(obj, "getMethod");
+    }
+
+    /**
+     * get http header keys
+     *
+     * @param obj HttpServletRequest
+     * @return key
+     */
+    public static Enumeration<?> getHeaderNames(Object obj) {
+        return (Enumeration<?>) ReflectUtils.invokeWithNoneParameter(obj, "getHeaderNames");
```

Comment:
Note that null is returned.

Reply:
in spring framework, headerNames are not null